l3kernel/l3file.dtx

```diff
@@ -917,6 +917,21 @@
 %   function. An error is raised if the file is not found.
 % \end{function}
 %
+% \begin{function}[added = 2023-05-18, EXP]{\file_input_raw:n}
+%   \begin{syntax}
+%     \cs{file_input_raw:n} \Arg{file name}
+%   \end{syntax}
+%   Searches for \meta{file name} in the path as detailed for
+%   \cs{file_if_exist:nTF}, and if found reads in the file as
+%   additional \TeX{} source. No data concerning the file is
+%   tracked. If the file is not found, no action is taken.
+%   \begin{texnote}
+%     This function is intended only for contexts where files must
+%     be read purely by expansion, for example at the start of a
+%     table cell in a \tn{halign}.
+%   \end{texnote}
```

Comment:
In particular, it omits the execution of all file hooks.

or something like that. Also: shouldn't it be  "an \tn{halign}" ?

Reply:
File hooks are LaTeX2e things so I think for expl3 functions there're always no file hooks involved?

Reply:
I thin the h' in `halign` is voiced, so 'a' not 'an' (so like 'hotel' not 'hour')

Reply:
Not around my manor it ain't!  Look it up??:-)

It is in fact neither voiced nor unvoiced as, in my part of TeXland, it is not even pronounced!--
halign is voiced as aitch-align
hence "an halign", please!

Reply:
Not around my manor it ain't!  Look it up??:-)

It is in fact neither voiced nor unvoiced as, in my part of TeXland, it is not even pronounced!--
halign is voiced as aitch-align
hence "an halign", please!

Reply:
Joseph, given his profession, may be confusing it "haline"??